mis_builder/models/mis_report.py

```diff
@@ -74,7 +74,7 @@ class MisReportKpi(models.Model):
 
     name = fields.Char(size=32, required=True,
                        string='Name')
-    description = fields.Char(required=True,
+    description = fields.Char(required=False,
```

Comment:
Why changing this?

Reply:
misconfig after fix, now tested on True and working.
Thanks for review